frontend/src/components/BlockchainTreeView/BlockchainTreeView.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './BlockchainTreeView.css';
+import Tree from 'react-d3-tree';
+
+export interface BlockchainBlock {
+  hash: string;
+  children: BlockchainBlock[];
+}
```

Comment:
Bu interface `BlockchainData`nın aynısı. İkisinden birini silelim.

Not: İsmin `BlockchainBlock` olması bana daha mantıklı geliyor.